newIDE/app/public/index.html

```diff
@@ -61,5 +61,20 @@
         </p>
       </div>
     </noscript>
+    <script>
+      (function() {
+        // Apply background color for startup
+        var backgroundColors = {
+          Dark: '#252525',
+          Nord: '#2E3440',
+          'Solarized Dark': '#00212B'
+        };
```

Comment:
This will need to be added to whenever a new theme is contributed

Reply:
Include a comment in the place where themes are declared so that this information is not lost.